classes/xsct-tarball.bbclass

```diff
@@ -97,6 +97,7 @@ python xsct_event_extract() {
             mkdir -p ${XSCT_STAGING_DIR}; \
             cd ${XSCT_STAGING_DIR}; \
             tar -xvf ${XSCT_DLDIR}/${XSCT_TARBALL};")
+
```

Comment:
This is probably not meant to be in the commit. If you do want to propose this change, probably you want to make a better (different) commit headline. Don't forget to follow the commit headline convention which is to have a prefix for the location of your change. In this case, `xsct-tarball: ...`

Reply:
... this goes for the other commit in your MR. ;)